### ISSUE TITLE
Is it possible to have links in popovers?

### ISSUE THREAD

**Issue by baranelitez** [nathanreyes/v-calendar#411]
Hi. Is it possible to customize popovers of calendar such that it has links in it?

**Comment by nathanreyes** [comment]
Yes it is. I would utilize the [custom popover slot](https://vcalendar.io/guide/attributes.html#_2-scoped-slot) for this purpose.
